packages/wdio-cli/src/supportedPackages.js

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
should we move this into `constants.js` or do you see other stuff in this file in the future?